### DISCUSSION TITLE
Meeting: May 28, 2024 agenda

### DISCUSSION THREAD

**Discussion by earth2marsh** [OAI/sig-moonwalk#114] [Announcements]
## Weekly meetings happen on Tuesdays

**Time:** 5pm GMT / 9am Pacific
**Zoom:** [https://zoom.us/j/975841675](https://zoom.us/j/975841675?pwd=SUh4MjRLaEFKNlI3RElpWTdhRDVVUT09), dial-in passcode: 763054

### Accessibility & Etiquette

* Participants must abide by our [Code-of-Conduct](https://github.com/OAI/OpenAPI-Specification/blob/main/CODE_OF_CONDUCT.md#code-of-conduct).
* Meetings are recorded for future reference, and for those who are not able to attend in-person.
* We invited challenges to language or behaviour that is harmful or not inclusive during this meeting.
* We look forward to your participation, but please consider these acts of etiquette:
  * Remain on mute when not speaking to prevent interruptions.
  * Blur your background to reduce visual distractions.
  * Use the Zoom meeting "Raise Hand" feature to notify the group when you wish to speak.

| Blur My Background | Raise Hand |
|-|-|
| <img width="323" alt="Screenshot of Zoom UI showing the 'Stop Video' and 'Blur My Background' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/7e43dbbb-6529-46e6-8b04-4c1aa852d9dd"> | <img width="323" alt="Screenshot of Zoom UI showing the 'Reaction' and 'Raise Hand' control" src="https://github.com/OAI/OpenAPI-Specification/assets/7367/f991722f-4651-40aa-9bc4-7e9a2a165a6a"> |

### Agenda Structure

| Topic | Owner | Decision/NextStep |
|-------|---------|---------|  
| | | |
Intros and governance meta-topics (5 mins) | SIG members | -- |
General discussion topics (list below) | SIG members | -- |
Wrap-up and seed the next agenda (5 mins) | SIG members | -- |

Whether attending or not, **anyone can comment on this issue prior to the meeting to suggest topics or to add comments on planned topics or proposals**.

**Comment by earth2marsh** [comment]
@darrelmiller 's point in last week's chat may be a good place to start this week:
> perhaps we can have the the best of both worlds where having a default "signature definition" and then override it per "request".

In the interim, the [working document](https://docs.google.com/document/d/1OM9MuiuafpiX2R-cNIhvaf7xJH95sXH7JtsMS0erxfQ/edit) is a place to add thoughts asynchronously. One opportunity in that doc is to flesh out the table of contents that Gary began.

**Reply by handrews** [reply]
If there's additional time, I'd like to discuss what "modularity" means for Moonwalk.

**Comment by earth2marsh** [comment]
Today's wide-ranging, exploratory discussion of modularity is available in [video](https://zoom.us/rec/share/puOHItH7rJb4IhHdXHhcYPgvp90bS7qN8qrrnsdGutP2CUZ2huIPL9aP2PcGQELU.pLX3pO1bS1AOrPgb). Clearly, this is an important topic worthy of more discussion!

**Reply by ahx** [reply]
@earth2marsh The [link to the video](https://zoom.us/rec/play/WqbRX_l-mn-_Av1avw3hp7CLZKO204dmyTKkjFLqNaQRMirCMt-sCTiqsOtq8yOwvfZrIfTzwiZ4z4A5.LwrnxYXEZSlcETGv) is not correct. It's only 14 seconds long.

**Reply by earth2marsh** [reply]
Sorry! Should be fixed now.

**Reply by ahx** [reply]
Thanks and thanks for the work.
